apps/browser/qml/pages/BrowserPage.qml

```diff
@@ -343,7 +343,9 @@ Page {
             }
 
             webView.grabActivePage()
-            if (!webView.tabModel.activateTab(url)) {
+            if (webView.tabModel.activateTab(url)) {
+                webView.unownActiveTab()
```

Comment:
Whilst unown can be understood, maybe this should be called like releaseActiveTabOwnership or takeActiveTabOwnership.  Latter wouldn't actually "take" anything but would freeing already set ownership.

Summary: to me releasing / freeing feels quite natural.

Reply:
Renamed the method.